src/ccall.cpp

```diff
@@ -1686,7 +1695,11 @@ static jl_cgval_t emit_ccall(jl_codectx_t &ctx, jl_value_t **args, size_t nargs)
                 true);
         setName(ctx.emission_context, signal_page_load, "signal_page_load");
         ctx.builder.CreateBr(contBB);
+#if JL_LLVM_VERSION >= 160000
+        ctx.f->insert(ctx.f->end(), contBB);
```

Comment:
BasicBlock::insertInto will make all of these diffs cleaner (supported in 14.X too).